.travis.yml

```diff
@@ -3,10 +3,10 @@ python:
   - '3.6'
 
 install:
-  - pip install --upgrade nox-automation
+  - pip install nox-automation==0.19.1
```

Comment:
Going forward, it seems we only need:

```shell
pip install --upgrade nox
nox -f nox.py -s docs
```

Pin the tooling version has pros and cons, personally I'm leaning toward using the latest version.

Reply:
I agree, but that does mean making some changes to our tests and CI config. Any objection to merging this to unbreak the build and merging https://github.com/census-instrumentation/opencensus-python/pull/653 when it's working?